src/DonationForms/AsyncData/Actions/GetAsyncFormDataForListView.php

```diff
@@ -110,6 +111,8 @@ private function isFormGoalEnabled(int $formId): bool
         }
 
         return give_is_setting_enabled(Give()->form_meta->get_meta($formId,
-            DonationFormMetaKeys::GOAL_OPTION()->getKeyAsCamelCase(), true));
+            DonationFormMetaKeys::GOAL_OPTION,
```

Comment:
Ops! When I wrote this, I got confused because we are using the camel case approach for the object model: https://github.com/impress-org/givewp/blob/develop/src/DonationForms/V2/DataTransferObjects/DonationFormQueryData.php#L74

Reply:
All good man!